### ISSUE TITLE
Wrong external IDs being cached

### ISSUE THREAD

**Issue by juanma-cvega** [jurialmunkey/plugin.video.themoviedb.helper#254]
I just found an issue when trying to watch an episode that got its TVDB ID updated on TheMovieDB. I've tried watching the same episode on an Nvidia Shield and on a MacBook, the first one cannot find the episode on Embycon based on its TVDB ID, neither from the library, even after running the refresh info command, nor directly from the addon. I then tried to watch the episode on the MacBook and it works fine and I see in the logs the right TVDB ID being used. 
So my guess is that the ID got cached but then it was changed on TheMovieDB. So is there any way to trigger a cache refresh?

**Comment by drinfernoo** [comment]
Either from the context menu or by browsing to the show in the add-on, and choosing "Refresh Cache".

**Comment by juanma-cvega** [comment]
I just tried that and it's still not working so maybe the problem is something else. I'll try to grab the logs from the Nvidia Shield when I have the time to see which TVDB ID it's using to verify the ID is the issue.

**Comment by drinfernoo** [comment]
What show are you referring to?

**Comment by juanma-cvega** [comment]
Saint seiya

**Comment by jurialmunkey** [comment]
Possible that it is falling back to the IDs on Trakt (player will try to get IDs from TMDb first and then will try Trakt). However, weird that it is working on one device and not the other.

The details from TMDb are cached, but refreshing should work. Did you try refreshing cache for both the base tvshow and also the individual episode?

**Comment by juanma-cvega** [comment]
Well, I just found the issue and it was on my side. I was using a previous version of the Embycon player with the year in the query string. Saint Seiya is quite a mess, the way TVDB and TMDB handle it is completely different to the point not even the release dates are the same. So basically the release dates are some weeks apart so the episodes that fall at the end of the year have different air years and the query was failing because of that. That's why I created a pull request to remove the year from the play episode query but apparently I forgot to update my Shield. Sorry about that.
